opentasks-provider/build.gradle

```diff
@@ -17,11 +17,26 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    packagingOptions {
+        exclude 'META-INF/NOTICE'
+        exclude 'META-INF/LICENSE'
+    }
 }
 
 dependencies {
     compile('org.dmfs:rfc5545-datetime:0.2.4')
     compile('org.dmfs:lib-recur:0.9.6')
     compile project(':opentasks-contract')
-    testCompile 'junit:junit:4.12'
+
+    // TODO Review
+
+    androidTestCompile project(':opentaskspal')
+    androidTestCompile 'com.android.support:support-annotations:' + SUPPORT_LIBRARY_VERSION
+    androidTestCompile 'com.android.support.test:runner:0.5'
+    androidTestCompile 'com.android.support.test:rules:0.5'
+    androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
+
+    // https://github.com/amitshekhariitbhu/Android-Debug-Database
+    // Usage: "adb forward tcp:8080 tcp:8080" then open "http://localhost:8080"
+    androidTestCompile 'com.amitshekhar.android:debug-db:1.0.1'
```

Comment:
What's this for and does it have to be in master?

Reply:
This was very useful for me, it allows to check the content of the db from browser. Do you maybe use something else for that?
Anyway, I've removed it, no need for it to be committed.